src/ops.rs

```diff
@@ -307,7 +319,7 @@ impl KeyPair {
     /// ```
     #[allow(non_snake_case)]
     pub fn new(pw: &[u8], owner: String, d: &SecParam) -> Result<KeyPair, OperationError> {
-        let data = kmac_xof(&pw.to_vec(), &[], 448, "SK", d)?;
+        let data = kmac_xof(&pw.to_vec(), &[], 448, "SK", d)?; 
```

Comment:
run ```cargo fmt``` when you are pushing up changes. you dont want to be adding spaces like this